addons/mail/static/tests/core/record.test.js

```diff
@@ -960,6 +964,8 @@ test("record.toData() is JSON stringified and can be reinserted as record", asyn
     expect(p2.team.name).toBe("Discuss");
     expect(toRaw(store.Person.records[p2.localId])).toBe(toRaw(p2));
     expect(serializeDateTime(p2.due_datetime)).toBe("2024-08-28 10:19:44");
+    expect(p2.signature.toString()).toBe("<p>-- John</p>");
+    expect(p.signature).toBeInstanceOf(Markup);
```

Comment:
This test is good for the feature, but in terms of security, there is missing the inverse case: ensuring that non-markup stays non-markup.

Reply:
Inserting data in html field will is inserted as `htmlEscape(value)` so that's necessarily a markup. So in practice we can't have a non-markup content in an html field so we can't test that scenario.